packages/nc-gui-v2/components/smartsheet/sidebar/toolbar/ToggleDrawer.vue

```diff
@@ -0,0 +1,18 @@
+<script setup lang="ts">
+/** Sidebar visible */
+const { isOpen, toggle } = useSidebar({ storageKey: 'nc-right-sidebar' })
+</script>
+
+<template>
+  <a-tooltip :placement="isOpen ? 'bottomRight' : 'left'">
```

Comment:
Can we put a debounce on the tooltip? like :mouse-enter-delay="0.5" or sth, so it doesn't instantly pop up?